mmdeploy/codebase/mmdet3d/models/pillar_scatter.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmdet3d.models.middle_encoders.pillar_scatter.'
+    'PointPillarsScatter.forward_batch',
+    backend='tensorrt')
+def pointpillarsscatter__forward(ctx, self, voxel_features, coors, batch_size):
+    """Scatter features of single sample.
+
+    Args:
+        voxel_features (torch.Tensor): Voxel features in shape (N, M, C).
+        coors (torch.Tensor): Coordinates of each voxel.
+            The first column indicates the sample ID.
+        batch_size (int): Number of samples in the current batch.
+    """
+    canvas = torch.zeros(
+        self.in_channels,
+        self.nx * self.ny,
+        dtype=voxel_features.dtype,
+        device=voxel_features.device)
+
+    indices = coors[:, 2] * self.nx + coors[:, 3]
```

Comment:
This is different from https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/middle_encoders/pillar_scatter.py . which is `indices = coors[:, 1] * self.nx + coors[:, 2]`

Reply:
Here func input is different.